.github/workflows/build-package.yml

```diff
@@ -0,0 +1,59 @@
+name: Build and Push Docker Image
+
+on:
+  push:
+    branches: ['main']
+  pull_request:  
```

Comment:
Do we want to create a release on every pull request or only when we specifically take an action to bless a container version?

Reply:
A release isn't created. The release steps are conditioned out on non-main branches